lib/gcloud/bigquery/connection.rb

```diff
@@ -462,14 +462,14 @@ def copy_table_config source, target, options = {}
           "configuration" => {
             "copy" => {
               "sourceTable" => {
```

Comment:
Hmmm, good point. This method could be seen as "to gapi tableReference". Should I change it to camel case keys?